addons/repair/models/repair.py

```diff
@@ -164,6 +164,8 @@ def _default_picking_type_id(self):
         help="True if this repair is linked to a Return Order and the order is 'Done'. False otherwise.")
     picking_product_ids = fields.One2many('product.product', compute='compute_picking_product_ids')
     picking_product_id = fields.Many2one(related="picking_id.product_id")
+    picking_lot_ids = fields.One2many('stock.lot', compute='compute_picking_lot_ids')
+    picking_lot_id = fields.Many2one(related="picking_id.lot_id")
```

Comment:
Both seem redundant.
* `picking_lot_id` = `picking_id.move_line_ids.lot_id`
* `picking_lot_ids` = `repair.picking_id.move_ids.lot_ids`

I think you can only keep `picking_lot_ids` for the domain